### PR TITLE
[SOAR-18025] Cloudflare - Snyk Vulnerability, SDK Bump and Unit test fix

### PR DESCRIPTION
**Pull request by rmurray-r7**
### Description

Ticket: https://rapid7.atlassian.net/browse/SOAR-18025

Snyk: https://app.snyk.io/org/soar-insightconnect-new/project/9f398923-2b9f-4d03-8d77-2220b9b4776c

Describe the proposed changes:

  - SDK Bump to 6.1.4
  - Bumped requirements.txt
  - Fixed unit test to relfect changes in `ipv4/6_cidr`

## PR Requirements

Developers, verify you have completed the following items by checking them off:

### Testing

Tested snyk internally and there are no longer vulnerabilities:

image

#### Unit Tests

After fixing the paths/unit tests, they all pass:

image

### CODE REVIEW

File: plugins/cloudflare/requirements.txt
Comment (igorski-r7) [inline plugins/cloudflare/requirements.txt:1]: Could you please add new line at the end of file? It's like Github is marking this 😄

**Review by igorski-r7 — APPROVED** [review]

File: plugins/cloudflare/icon_cloudflare/util/helpers.py [hunk 1]
```diff
@@ -51,12 +51,12 @@ def clean(item_to_clean: Union[dict, list]) -> Union[dict, list]:
 
 
 def set_configuration(target: str) -> dict:
-    if validators.ipv4(target):
+    if validators.ipv4(target, cidr=False):
         return {"value": target, "target": "ip"}
-    if validators.ipv6(target):
+    if validators.ipv6(target, cidr=False):
         return {"value": target, "target": "ip6"}
```
Comment (cmcnally-r7) [inline plugins/cloudflare/icon_cloudflare/util/helpers.py:57]: when was this plugin last updated? I remember these changes being needed for this validators package, but just want to double check these `cidr=False` values don't break anything, or if they were the default for these methods before.

File: plugins/cloudflare/plugin.spec.yaml [hunk 2]
```diff
@@ -4,16 +4,45 @@ products: [insightconnect]
...
+  - "`Firewall Services:Read`"
+  - "`Firewall Services:Edit`"
+  - "`Zone:Read`"
+version_history:
+  - "1.0.1 - Bumping requirements.txt | SDK bump to 6.1.4"
+  - "1.0.0 - Initial plugin | Add Get Accounts, Get Zones, Get Lists, Get Zone Access Rules, Create Zone Access Rule and Delete Zone Access Rule actions"
+links:
+  - "[Cloudflare](https://www.cloudflare.com)"
+references:
+  - "[Cloudflare](https://www.cloudflare.com)"
```
Comment (cmcnally-r7) [inline plugins/cloudflare/plugin.spec.yaml:45]: Can we get some cloudflare api docs in here instead

**Review by cmcnally-r7 — CHANGES_REQUESTED** [review]

File: plugins/cloudflare/icon_cloudflare/util/helpers.py (hunk 1 above)
Comment (rmurray-r7) [reply plugins/cloudflare/icon_cloudflare/util/helpers.py:57]: It was last updated a little over a year ago. The false values are only used to recognise non cidr ip addresses, as the cidr function has been removed from the validator package. I tested them out and everything works perfect! 😄

**Review by cmcnally-r7 — APPROVED** [review]

**Pull request merged**